Common_Engine/Common_Engine.csproj

```diff
@@ -39,6 +39,10 @@
       <HintPath>..\..\BHoM\Build\Common_oM.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="Dimensional_oM, Version=3.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Reference should not contain version, culture, or processorArchitecture.
Reference should also have `copy local` set to false.